newIDE/app/src/SceneEditor/SetupGridDialog.js

```diff
@@ -43,6 +48,18 @@ export default class SetupGridDialog extends Component {
         open={this.props.open}
         onRequestClose={this.props.onCancel}
       >
+        <MiniToolbar>
+          <MiniToolbarText>
+            <Trans>Line color:</Trans>
+          </MiniToolbarText>
+          <ColorPicker
```

Comment:
I think that here we'll avoid the MiniToolbar entirely, as it's a dialog containing fields.
Speaking of fields, can you try using `ColorField`? Might go better with the other TextField of this dialog.

Reply:
(Move the MiniToolbarText to the `floatingLabelText` prop of `ColorField`)